packages/host/src/DetailPanel.test.tsx

```diff
@@ -0,0 +1,93 @@
+import { render } from '@testing-library/react';
+import React, { FC, useEffect } from 'react';
+import { act } from 'react-dom/test-utils';
+import {
+  Action,
+  IntlTestProvider,
+  Section,
+  useDetailPanel,
+} from '@openmsupply-client/common';
+import DetailPanel from './DetailPanel';
+
+describe('DetailPanel', () => {
+  const DetailPanelExample: FC<{ sections: Section[]; actions: Action[] }> = ({
+    sections,
+    actions,
+  }) => {
+    const { OpenButton, setActions, setSections } = useDetailPanel();
+
+    useEffect(() => setActions(actions), []);
+    useEffect(() => setSections(sections), []);
+
+    return (
+      <IntlTestProvider locale="en">
+        <div>{OpenButton}</div>
+        <DetailPanel />
+      </IntlTestProvider>
+    );
+  };
+
+  it('Does not render empty panel', () => {
+    const actions: Action[] = [];
+    const sections: Section[] = [];
+    const { queryByText, queryByTestId } = render(
+      <DetailPanelExample actions={actions} sections={sections} />
+    );
+
+    act(() => {
+      queryByText('More')?.click();
```

Comment:
🤔  the screen by default is large enough that the drawer is automatically open. So the `queryByText`'s always can't find anything and the click doesn't happen - are they meant to be ensuring that if the test environment screen changes the test still passes?

Also the drawer is always rendered except when actions/sections aren't passed, it just has nothing in it so is hidden - `toBeVisible` doesn't seem to work .. best I came up with was `toHaveAttribute("aria-expanded, "false")`